src/gmusic-theme.js

```diff
@@ -74,6 +96,7 @@ window.GMusicTheme = class GMusicTheme {
     this.BACK_HIGHLIGHT = colorObject.backHighlight || this.BACK_HIGHLIGHT;
     this.FORE_PRIMARY = colorObject.forePrimary || this.FORE_PRIMARY;
     this.FORE_SECONDARY = colorObject.foreSecondary || this.FORE_SECONDARY;
+    this.type = colorObject.type || this.type;
```

Comment:
We should probably add an assertion for `colorObject.type` to be kind to developers who got a typo (or miss the deprecation notice for accent when we introduce it):

```js
static TYPES = {
  FULL: 'FULL',
  HIGHLIGHT_ONLY: 'HIGHLIGHT_ONLY',
};
static VALID_TYPES = [TYPES.FULL, TYPES.HIGHLIGHT_ONLY];

updateTheme(colorObject) {
  if (colorObject.type !== undefined && GMusicTheme.VALID_TYPES.indexOf(colorObject.type) !== -1) {
    throw new Error(`\`updateTheme\` expected \`colorObject.type\` to be in ${JSON.stringify(GMusicTheme.VALID_TYPES)} ` + 
      `but it was "${colorObject.type}"`);
  }
```